<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnInicio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-color: qlineargradient(spread:pad, x1:0.522409, y1:1, x2:0.472, y2:0, stop:0 rgba(112, 40, 161, 255), stop:1 rgba(156, 60, 225, 255));*/

background-color: qlineargradient(spread:pad, x1:0, y1:0.528, x2:1, y2:0.551136, stop:0 rgba(92, 106, 155, 255), stop:1 rgba(60, 69, 104, 255));

border:none;
/*color: #3e0c5f;*/
color: #ffffff;</string>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDetener">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-color: qlineargradient(spread:pad, x1:0.522409, y1:1, x2:0.472, y2:0, stop:0 rgba(112, 40, 161, 255), stop:1 rgba(156, 60, 225, 255));*/

background-color: qlineargradient(spread:pad, x1:0, y1:0.528, x2:1, y2:0.551136, stop:0 rgba(92, 106, 155, 255), stop:1 rgba(60, 69, 104, 255));


border:none;
/*color: #3e0c5f;*/
color: #ffffff;</string>
    </property>
    <property name="text">
     <string>Detener</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSalEstado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblEstado">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">Color:#FFFFFF;</string>
    </property>
    <property name="text">
     <string>Estado: Inicio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
</string>
    </property>
    <property name="text">
     <string>Entrada</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
</string>
    </property>
    <property name="text">
     <string>Traducci√≥n</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="SalidaTraducida">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>950</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0,255,0)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TextoPredicho">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>950</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0,255,0)</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxMicros">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>170</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="lblMicroInput">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>351</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Seleccione la entrada</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSeleccionEntrada">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>170</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-color: qlineargradient(spread:pad, x1:0.522409, y1:1, x2:0.472, y2:0, stop:0 rgba(112, 40, 161, 255), stop:1 rgba(156, 60, 225, 255));*/

background-color: qlineargradient(spread:pad, x1:0, y1:0.528, x2:1, y2:0.551136, stop:0 rgba(92, 106, 155, 255), stop:1 rgba(60, 69, 104, 255));

border:none;
/*color: #3e0c5f;*/
color: #ffffff;</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.652682, x2:0, y2:0.222, stop:0 rgba(81, 91, 126, 255), stop:1 rgba(132, 143, 175, 255));

/*background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(238, 112, 255, 255), stop:1 rgba(251, 231, 255, 255));*/</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>btnInicio</zorder>
   <zorder>btnDetener</zorder>
   <zorder>lblSalEstado</zorder>
   <zorder>lblEstado</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>SalidaTraducida</zorder>
   <zorder>TextoPredicho</zorder>
   <zorder>comboBoxMicros</zorder>
   <zorder>lblMicroInput</zorder>
   <zorder>btnSeleccionEntrada</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionIniciar"/>
    <addaction name="actionDetener"/>
    <addaction name="separator"/>
    <addaction name="actionLista_de_palabras_funables"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionTexto"/>
    <addaction name="actionColor_Texto"/>
    <addaction name="actionCambiar_Traducci_n"/>
    <addaction name="separator"/>
    <addaction name="actionFondo"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionIniciar">
   <property name="text">
    <string>Iniciar</string>
   </property>
  </action>
  <action name="actionTexto">
   <property name="text">
    <string>Texto</string>
   </property>
  </action>
  <action name="actionFondo">
   <property name="text">
    <string>Fondo</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir </string>
   </property>
  </action>
  <action name="actionDetener">
   <property name="text">
    <string>Detener</string>
   </property>
  </action>
  <action name="actionCambiar_Traducci_n">
   <property name="text">
    <string>Cambiar Traducci√≥n</string>
   </property>
  </action>
  <action name="actionColor_Texto">
   <property name="text">
    <string>Color Texto</string>
   </property>
  </action>
  <action name="actionLista_de_palabras_funables">
   <property name="text">
    <string>Lista de palabras funables</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
